app/views/candidate_interface/personal_details/show.html.erb

```diff
@@ -22,6 +22,51 @@
           <% end %>
         </dd>
       </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          <%= t('application_form.personal_details.date_of_birth.label') %>
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @personal_details_form.date_of_birth_format %>
+        </dd>
+
+        <dd class="govuk-summary-list__actions">
+          <%= link_to candidate_interface_personal_details_edit_path, class: 'govuk-link' do %>
+            Change<span class="govuk-visually-hidden"> date of birth</span>
+          <% end %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          <%= t('application_form.personal_details.nationality.label') %>
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @personal_details_form.nationalities[0] + ' and ' + @personal_details_form.nationalities[1] %>
```

Comment:
Array indices in views make me nervous. Could this be in the form/a presenter (presumably there's logic to add around showing single or dual nationality)?

Reply:
Would `<%= @personal_details_form.nationalities.to_sentence %>` work here?